### PR TITLE
[1LP][RFR] new test_tag_change_tag_name

### PR DESCRIPTION
**Pull request by lina-nikiforova**
{{ pytest: -v cfme/tests/configure/test_tag.py::test_updated_tag_name_on_vm }}

also moved tag creation from `test_tag_crud` to fixture to use it in the new test

### CODE REVIEW

File: cfme/tests/configure/test_tag.py [hunk 1]
```diff
@@ -57,22 +57,27 @@ def category(appliance):
...
 
 
+@pytest.fixture
+def tag(category):
+    tag = category.collections.tags.create(
+        name=fauxfactory.gen_alphanumeric(8).lower(),
+        display_name=fauxfactory.gen_alphanumeric(32)
+    )
+    yield tag
+    if tag.exists:
```
Comment (valaparthvi) [inline cfme/tests/configure/test_tag.py]: You can use `delete_if_exists` method instead.

File: cfme/tests/configure/test_tag.py [hunk 2]
```diff
@@ -107,6 +112,52 @@ def test_map_tagging_crud(appliance, category, soft_assert):
                                       .format(map_tag_entity.label))
 
 
+def test_tag_change_tag_name(provider, tag, request):
```
Comment (sshveta) [inline cfme/tests/configure/test_tag.py]: Can you change the test name to  "test_updated_tag_name_on_vm"

File: cfme/configure/configuration/region_settings.py [hunk 3]
```diff
@@ -87,6 +87,17 @@ def delete(self, cancel=False):
         view.browser.handle_alert(cancel=cancel)
         view.flash.assert_no_error()
 
+    def change_name(self, name):
```
Comment (sshveta) [inline cfme/configure/configuration/region_settings.py]: Optional : I think we should rename this method to "update_tag_name"

File: cfme/tests/configure/test_tag.py [hunk 4]
```diff
@@ -107,6 +112,52 @@ def test_map_tagging_crud(appliance, category, soft_assert):
...
+    tag.change_name(new_tag_name)
+
+    vm_tags = vm.get_tags()
+
+    # assert the tag was not changed in the UI
+    assert any(
+        tag.category.display_name == vm_tag.category.display_name and
+        tag.display_name == vm_tag.display_name
+        for vm_tag in vm_tags
+    ), 'tag is not assigned'
```
Comment (sshveta) [inline cfme/tests/configure/test_tag.py]: BZ also states that vm should not show the tag when the tag is deleted .
Can we include that check too here ?

File: cfme/tests/configure/test_tag.py (hunk 2 above)
Comment (lina-nikiforova) [reply cfme/tests/configure/test_tag.py]: @sshveta ok, makes sense

File: cfme/tests/configure/test_tag.py (hunk 4 above)
Comment (lina-nikiforova) [reply cfme/tests/configure/test_tag.py]: @sshveta I don't see this in the BZ. The tag disappeared in the UI after the tag's name was changed, but it was a bug and was observer before the fix.

File: cfme/tests/configure/test_tag.py [hunk 5]
```diff
@@ -57,22 +57,27 @@ def category(appliance):
...
         casecomponent: Tagging
     """
-    tag = category.collections.tags.create(
-        name=fauxfactory.gen_alphanumeric(8).lower(),
-        display_name=fauxfactory.gen_alphanumeric(32).lower()
-    )
-    with update(tag):
-        tag.display_name = fauxfactory.gen_alphanumeric(32).lower()
-    tag.delete()
+    assert tag.exists
```
[the remark this answers is not in the document]
Comment (lina-nikiforova) [reply cfme/tests/configure/test_tag.py]: @sshveta now I realized what was supposed to happen here, update didn't work as expected, it created the new tag in the UI, so now I'm updating the method itself. Thanks

File: cfme/configure/configuration/region_settings.py (hunk 3 above)
Comment (lina-nikiforova) [reply cfme/configure/configuration/region_settings.py]: @sshveta I removed this method as I realized that `update` method doesn't do what it is supposed to do. Now I fixed it and can utilize it in this test as well.

**Pull request merged**